uint16 bus
uint16 address
---
bool hasSucceeded
int32 handle